pages/linux/picom.md

```diff
@@ -0,0 +1,16 @@
+# picom
+
+> Standalone compositor for Xorg.
+> More information: <https://wiki.archlinux.org/title/picom>.
```

Comment:
I generally would have suggested linking to the [project](https://github.com/yshui/picom), but Arch wiki seems to have better documentation here, so IG this link is fine.